README.rst

```diff
@@ -360,6 +360,10 @@ Choose from the list of available fixers:
                         Remove trailing commas in list
                         function calls.
 
+* **lower_case_cast** [symfony]
```

Comment:
:+1:  done in https://github.com/SpacePossum/PHP-CS-Fixer/commit/de2705df1abfc2670967e002d79a9ce188e706c2#diff-88b99bb28683bd5b7e3a204826ead112L363